rs.py

```diff
@@ -44,4 +44,4 @@ def classify():
 
 if __name__ == '__main__':
     configure_service('resources/config/app_config.cfg')
```

Comment:
`app_config.cfg` contains variables for the classifiers:

```
DEBUG = False
INNER_PKL = 'resources/classifiers/inner/inner_classifier.pkl'
MIDDLE_PKL = 'resources/classifiers/middle/middle_classifier.pkl'
OUTER_PKL = 'resources/classifiers/outer/outer_classifier.pkl'
MASTOID_PKL = 'resources/classifiers/mastoid/mastoid_classifier.pkl'
```

Should these be moved to a different file?